app/views/layouts/mailer.html.erb

```diff
@@ -289,7 +290,22 @@
 									<td>
 										<table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="30" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
 
-										<div class="h3-2-center" style="color:#1e1e1e; font-family:Arial, sans-serif; min-width:auto !important; font-size:20px; line-height:26px; text-align:center; letter-spacing:5px">RUBYGEMS&#65279.ORG</div>
+										<table width="100%" border="0" cellspacing="0" cellpadding="0">
+										  <tr>
+										    <td align="center">
+										        <table border="0" cellspacing="0" cellpadding="0">
+										          <tr>
+										            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
+										              <img src=<%= image_url("rubygems_logo_black.png") %> border="0" width="40" height="40" alt="" />
```

Comment:
This is fixed by most email client but src attr value should be in quotes

``` diff
--- app/views/layouts/mailer.html.erb
+++ app/views/layouts/mailer.html.erb
@@ -296,7 +296,7 @@
                                                                                        <table border="0" cellspacing="0" cellpadding="0">
                                                                                          <tr>
                                                                                            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
-                                                                                             <img src=<%= image_url("rubygems_logo_black.png") %> border="0" width="40" height="40" alt="" />
+                                                                                             <img src="<%= image_url("rubygems_logo_black.png") %>" border="0" width="40" height="40" alt="" />
                                                                                            </td>
                                                                                            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
                                                                                              <div class="h3-2-center" style="color:#1e1e1e; font-family:Arial, sans-serif; min-width:auto !important; font-size:25px; line-height:46px; text-align:center; letter-spacing:5px">RUBYGEMS&#65279.ORG</div>
```

Reply:
Thanks :+1: Updated to used double quotes in all src values.